package.json

```diff
@@ -27,6 +27,7 @@
   "homepage": "https://github.com/kschmitz123/boilerplate#readme",
   "devDependencies": {
     "concurrently": "^5.3.0",
+    "dotenv": "^8.2.0",
```

Comment:
`dotenv` is a dependency because it's required by your source code. Your server would crash, if it's not installed.